resources/metadata/whitesource.yaml

```diff
@@ -372,16 +372,16 @@ spec:
       - STAGES
       - STEPS
       default: -1
-    - name: includes
+    - name: include
```

Comment:
shouldn't we stick with `includes`? This is in line with stashing on Jenkins

Reply:
I was trying to stay consistent with Fortify's exclude flag, should we change that too?

Reply:
you are right, we are inconsistent here ...
In addition Fortify uses a list and here we have a string ...

The Jenkins stash using `includes` behaves exactly like this one here with a string.
I would suggest to stick with it.

@nevskrem, @CCFenner  what do you think?